index.html

```diff
@@ -27,14 +27,20 @@
       <center>
         <h1><i class="fa fa-file-pdf-o"></i></h1>
         <h1>Bookletize.js</h1>
-	<p style="width:50%;">Reformat PDFs into booklets in 4,1,2,3 or 8,1,2,7/6,3,4,5 format in JavaScript for making bound paper booklets. Print the resulting PDF, fold in half and staple to form a booklet. <br /> <a href="https://github.com/jywarren/bookletize.js">Open Source</a> by <a href="https://unterbahn.com">Jeffrey Yoo Warren</a></p>
+	<p style="width:50%;">JavaScript widget that reformats PDFs into signatures for making bound paper booklets. Print the resulting PDF, fold the stack in half and staple along the spine to form a <b>saddle stitch booklet</b>, or fold and stack sheets individually then glue the spine for a <b>perfect binding</b>.</p>
+  <p style="width:50%;"><a href="https://github.com/jywarren/bookletize.js">Open Source</a> by <a href="https://unterbahn.com">Jeffrey Yoo Warren</a></p>
+  <h2>Binding</h2>
```

Comment:
```suggestion
  <h2>Binding <a href="https://github.com/jywarren/bookletize.js/pull/3"><i class="fa fa-question-circle-o" style="font-size:11pt;"></i></a></h2>
```

I made a small change here to include a link back to this PR for credit/thanks/further explanation. Thanks!

<img width="853" alt="Screenshot 2025-01-07 at 9 30 47 AM" src="https://github.com/user-attachments/assets/d841c408-2d5b-45e0-93a4-e277bc0cdb39" />

Reply:
Oops, this didn't get added in, i must've forgotten to accept the change. I'll re-add it.

Reply:
https://github.com/jywarren/bookletize.js/pull/4